example/tile_properties.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
this sample should be included as godoc comment for `GetTilesetTile` function

Reply:
I'm sorry I don't understand exactly what you mean by this. I guess my understanding of Godoc is lacking.
How do I link to that code in the godoc comment?

Reply:
According to the godoc documentation examples functions should be named Example{FuncName} and be placed in the same package in a file ending with _test

This is not how the tmx2img example was done so I just did it the same way and I cannot find a reference to the tmx2img example anywhere in the comments so I don't know what else do to.

Reply:
tmx2img is more like example usable tool. Code examples should be added like this:
https://dev.to/slomek/testable-examples-in-go--1f0k